### PR TITLE
Updated integration tests

### PR DESCRIPTION
**Pull request by carlbm**

### CODE REVIEW

File: src/Common/SFA.DAS.Payments.DCFS/SFA.DAS.Payments.DCFS/Infrastructure/Data/DcfsRepository.cs [hunk 1]
```diff
@@ -28,6 +28,10 @@ protected T[] Query<T>(string command, object param = null, int timeout = 180)
                             commandTimeout: timeout)
                         .ToArray();
                 }
+                catch (SqlException e)
```
Comment (Pauljgraham) [inline src/Common/SFA.DAS.Payments.DCFS/SFA.DAS.Payments.DCFS/Infrastructure/Data/DcfsRepository.cs:31]: This adds no value, why bother putting it in?

**Review by Pauljgraham — APPROVED** [review]
just one small  comment

File: src/Common/SFA.DAS.Payments.DCFS/SFA.DAS.Payments.DCFS/Infrastructure/Data/DcfsRepository.cs (hunk 1 above)
Comment (scottywakefield) [reply src/Common/SFA.DAS.Payments.DCFS/SFA.DAS.Payments.DCFS/Infrastructure/Data/DcfsRepository.cs:31]: I agree with that.

**Review by scottywakefield — CHANGES_REQUESTED** [review]

File: src/Common/SFA.DAS.Payments.DCFS/SFA.DAS.Payments.DCFS/Infrastructure/Data/DcfsRepository.cs (hunk 1 above)
Comment (carlbm) [reply src/Common/SFA.DAS.Payments.DCFS/SFA.DAS.Payments.DCFS/Infrastructure/Data/DcfsRepository.cs:31]: So that you can put a breakpoint on it...

**Pull request merged**